build/Program.cs

```diff
@@ -86,8 +86,8 @@ internal static class Program
 
         Target("watch", DependsOn("clean-dist"), async () => {
 
-            if (Environment.OSVersion.Platform != PlatformID.Win32NT) {
-                throw new WithoutStackException("Use dotnet watch + webpack watch without build system");
+            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && RuntimeInformation.IsOSPlatform(OSPlatform.OSX)) {
```

Comment:
I don't get it = `if windows _AND_ OSX -> throw exception`.
Originally the exception was thrown when the OS IS NOT WINDOWS

Reply:
my bad. fixed